src/shared/database/index.ts

```diff
@@ -270,22 +270,36 @@ async function handleRevocations(
   manifest: DatabaseManifest,
 ): Promise<DatabaseManifest> {
   // Mark all older manifests as revoked
-  query('manifests')
+  const revokedSigningKeys = await query('manifests')
```

Comment:
How often does HandleRevocations run?

Reply:
It gets called whenever a manifest is saved.